### PR TITLE
add app, controller and model files, set up get /api/topics

### PR DESCRIPTION
**Pull request by markielaz**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      .get("/api/topicz")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid Path");
+      });
+  });
+});
+
+describe("GET /api/topics", () => {
+  it("200: responds with a body of topics", () => {
```
Comment (karlaferry) [inline __tests__/app.test.js:27]: You want to also check the length of the topics array.

**Review by karlaferry — APPROVED** [review]
Approving this but please do add an assertion that checks the length of the topics array. :) Then feel free to merge when it's done!

File: __tests__/app.test.js (hunk 1 above)
Comment (markielaz) [reply __tests__/app.test.js:27]: thanks @karlaferry - could you check it again now I've added that to the test please?

Comment (karlaferry) [reply __tests__/app.test.js:27]: @markielaz Perfect! :)

**Pull request merged**